pkcs11/src/new/types/mechanism/mod.rs

```diff
@@ -1,704 +1,76 @@
 mod rsa;
 
 use crate::new::Error;
+use log::error;
 use pkcs11_sys::*;
 use std::convert::TryFrom;
+use std::ops::Deref;
 use std::ptr::null_mut;
 
+const RSA_PKCS_KEY_PAIR_GEN: MechanismType = MechanismType {
+    val: CKM_RSA_PKCS_KEY_PAIR_GEN,
+};
+const RSA_PKCS: MechanismType = MechanismType { val: CKM_RSA_PKCS };
```

Comment:
Are you moving these inside `MechanismType` as well?

Reply:
Yes I will in next PR where I implement the rest of the mechanisms